src/index.html

```diff
@@ -11,7 +11,108 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form>
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="personal-information">
+          Surname:
+          <input type="text" name="surname" 
+          autocomplete="off"
+          >
+        </label>
+        <label class="personal-information">
+          Name:
+          <input type="text" name="name" 
+          autocomplete="off"
+          >
+        </label>
```

Comment:
1. Try to implement this checkpoint throughout you code
![image](https://user-images.githubusercontent.com/122989585/218303157-c920b7f6-6844-4f4f-bb78-638ba5dfc222.png)
2. Add some space between large list items
```suggestion
      <fieldset>
        <legend>Personal information</legend>
        
        <label class="personal-information">
          Surname:
          <input 
            type="text" 
            name="surname" 
            autocomplete="off"
          >
        </label>
        
        <label class="personal-information">
          Name:
          <input 
            type="text" 
            name="name" 
            autocomplete="off"
          >
        </label>
```